common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,127 @@
+"""
+OLI analytics service event tracker backend.
+"""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from requests_oauthlib import OAuth1Session
+from urlparse import urljoin
+
```

Comment:
This empty line should be removed, so that `requests_oauthlib` and `django.*` appear together in the same stanza; they are both 3rd party installed packages.